src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -83,10 +82,12 @@ def get_attrs(self, item_list, user, **kwargs):
         for rule_activity in rule_activities:
             rpc_user = user_by_user_id.get(rule_activity.user_id)
```

Comment:
why do we have to use the rpc_user at all here? are alert rules in control or region silos?

Reply:
alert rules are in the region silo

Reply:
also hc was the one who added this stuff

Reply:
oh interesting.